modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/util/OperationUtils.java

```diff
@@ -413,9 +410,10 @@ public void UpdateUriOperations(SyncmlDocument syncmlDocument) throws DeviceMana
                             nmService.addNotification(notification);
```

Comment:
check if (nmService != null) before any method calls to avoid null pointer exceptions.